dev/Tiltfile

```diff
@@ -343,6 +351,18 @@ if to_run != []:
       enabled_resources.append(svc)
     config.set_enabled_resources(enabled_resources)
 
+to_test = cfg.get("test", [])
+if to_test != []:
+    if not is_ci:
+        print("Please re-run tests in 'ci' mode")
+    else:
+        enabled_resources = [
+           TEST_RESOURCES.get(label)
+           for label in to_test
+           if TEST_RESOURCES.get(label)
```

Comment:
is this indentation right?

Reply:
Refactored to be more readables (and less pythonic) https://github.com/GaloyMoney/galoy/pull/3615/commits/7ac47dce93177c85f6847364cd35488826a99529